lang/en/campaigns.php

```diff
@@ -480,4 +481,59 @@
         'public'    => 'Public campaign',
         'review'    => 'Awaiting Review',
     ],
+    'webhooks'              => [
```

Comment:
Please move it all to a new file `campaigns/webhooks.php` to keep translation files smaller